CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # libhoney Changelog
 
+## 1.15.1 2021-01-14
+
+### Improvements
+
+- Fix data race on dynFields length in Builder.Clone (#72)
+
+### Maintenance
+
+- Update dependencies
+    - github.com/klauspost/compress from 1.11.3 to 1.11.4
```

Comment:
We've had a couple of updates since the last release, v1.11.2 -> v1.11.3 -> v1.11.4. It's also nice to include PR links as a standard 👍 

```suggestion
    - github.com/klauspost/compress from 1.11.2 to 1.11.4 (#105, #106)
```